web/images/toolbarButton-openFile.svg

```diff
@@ -1,4 +1,5 @@
-<!-- This Source Code Form is subject to the terms of the Mozilla Public
-   - License, v. 2.0. If a copy of the MPL was not distributed with this
-   - file, You can obtain one at http://mozilla.org/MPL/2.0/. -->
-<svg xmlns="http://www.w3.org/2000/svg" width="16" height="16" viewBox="0 0 16 16"><path d="M14.859 3.2a1.335 1.335 0 0 1-1.217.8H13v1h1v8H2V5h8V4h-.642a1.365 1.365 0 0 1-1.325-1.11L6.584 1.538A2 2 0 0 0 5.219 1H2a2 2 0 0 0-2 2v10a2 2 0 0 0 2 2h12a2 2 0 0 0 2-2V5a2 2 0 0 0-1.141-1.8zM2 3h3.219l1.072 1H2zm7.854-.146L11 1.707V8.5a.5.5 0 0 0 1 0V1.707l1.146 1.146a.5.5 0 1 0 .707-.707l-2-2a.5.5 0 0 0-.707 0l-2 2a.5.5 0 0 0 .707.707z"></path></svg>
\ No newline at end of file
+<?xml version="1.0" encoding="UTF-8"?>
+<svg fill="none" version="1.1" viewBox="0 0 16 16" xmlns="http://www.w3.org/2000/svg" xmlns:cc="http://creativecommons.org/ns#" xmlns:dc="http://purl.org/dc/elements/1.1/" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#">
```

Comment:
This doesn't look entirely correct, compared to all the other files, and probably explains why this one is "larger" in the GitHub preview; don't you want the following here as well?
```suggestion
<svg width="16" height="16" viewBox="0 0 16 16" fill="none" xmlns="http://www.w3.org/2000/svg">
```

Reply:
It doesn't seem that the update fully addressed this, as suggested, since the "header" of this file still contains a bunch of unnecessary stuff when compared to e.g. `web/images/toolbarButton-download.svg`.

---

Also, the updated `toolbarButton-menuArrow.svg` file is missing from the patch (it was present originally).

Reply:
Initially I got a 12x12 icon but I wrongly replaced it with the 16x16 one used for page-down.
I pinged the UX designer in charge of that stuff about that, but it was an holiday yesterday in US, so we should have one in today or tomorrow.